ldes-server-admin/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/admin/rest/converters/ViewHttpConverter.java

```diff
@@ -51,21 +49,17 @@ public List<MediaType> getSupportedMediaTypes() {
 
 	@Override
 	public ViewSpecification read(@NotNull Class<? extends ViewSpecification> clazz, @NotNull HttpInputMessage inputMessage)
-			throws IOException, HttpMessageNotReadableException {
+			throws HttpMessageNotReadableException {
 		throw new UnsupportedOperationException("Not supported to read a viewSpecification");
 	}
 
 	@Override
 	public void write(@NotNull ViewSpecification view, MediaType contentType, HttpOutputMessage outputMessage)
 			throws IOException, HttpMessageNotWritableException {
-		Lang rdfFormat = rdfModelConverter.getLang(contentType, FETCH);
+		Lang rdfFormat = rdfModelConverter.getLang(contentType, REST_ADMIN);
 		rdfModelConverter.checkLangForRelativeUrl(rdfFormat);
-		StringWriter outputStream = new StringWriter();
 		Model model = viewSpecificationConverter.modelFromView(view);
-
-		RDFDataMgr.write(outputStream, model, rdfFormat);
-
-		OutputStream body = outputMessage.getBody();
-		body.write(outputStream.toString().getBytes());
+		outputMessage.getHeaders().setContentType(MediaType.valueOf(rdfFormat.getHeaderString()));
```

Comment:
Can't you use the 'contentType' parameter that you receive here? Now you convert this param to a lang and then extract it from the lang and map it back to your MediaType that you started with..